MaxText/generate_param_only_checkpoint.py

```diff
@@ -40,35 +40,56 @@
 
 Transformer = models.Transformer
 
-def _possibly_unroll_params(config, training_state, training_state_annotations, mesh):
-  """ If input layers are scanned, and force_unroll is set,
-      return modify training_state and train_state_annotations to be "unrolled".
-      Otherwise do nothing."""
+
+def _possibly_unroll_params(
+    config, training_state, training_state_annotations, mesh
+):
+  """If input layers are scanned, and force_unroll is set,
+  return modify training_state and train_state_annotations to be "unrolled".
+  Otherwise do nothing."""
   if not config.scan_layers or not config.force_unroll:
     return
 
-  training_state_layers = training_state.params['params']['decoder']['layers']
-  training_state_annotations_layers = training_state_annotations.params['params']['decoder']['layers']
+  training_state_layers = training_state.params["params"]["decoder"]["layers"]
+  training_state_annotations_layers = training_state_annotations.params[
+      "params"
```

Comment:
This is a surprising pylint choice to me, I guess due to max line cutoff

Reply:
I am surprised too. Let me fix this manually.